packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/amplify-meta-updaters.ts

```diff
@@ -11,54 +11,54 @@ import { AuthParameters } from '../import/types';
  * @param context The amplify context
  * @param resultMetadata The metadata from the service selection prompt
  */
-export const getPostAddAuthMetaUpdater = (context: any, resultMetadata: { service: string; providerName: string }) => (
-  resourceName: string,
-): string => {
-  const options: any = {
-    service: resultMetadata.service,
-    providerPlugin: resultMetadata.providerName,
-  };
-  const parametersJSONPath = path.join(context.amplify.pathManager.getBackendDirPath(), 'auth', resourceName, 'parameters.json');
-  const authParameters = JSONUtilities.readJson<AuthParameters>(parametersJSONPath)!;
-
-  if (authParameters.dependsOn) {
-    options.dependsOn = authParameters.dependsOn;
-  }
-
-  let customAuthConfigured = false;
-  if (authParameters.triggers) {
-    const triggers = JSONUtilities.parse<any>(authParameters.triggers);
-
-    customAuthConfigured =
-      !!triggers.DefineAuthChallenge &&
-      triggers.DefineAuthChallenge.length > 0 &&
-      !!triggers.CreateAuthChallenge &&
-      triggers.CreateAuthChallenge.length > 0 &&
-      !!triggers.VerifyAuthChallengeResponse &&
-      triggers.VerifyAuthChallengeResponse.length > 0;
-  }
-
-  options.customAuth = customAuthConfigured;
-  options.frontendAuthConfig = getFrontendConfig(authParameters);
+export const getPostAddAuthMetaUpdater =
+  (context: any, resultMetadata: { service: string; providerName: string }) =>
+  (resourceName: string): string => {
+    const options: any = {
+      service: resultMetadata.service,
+      providerPlugin: resultMetadata.providerName,
+    };
+    const parametersJSONPath = path.join(context.amplify.pathManager.getBackendDirPath(), 'auth', resourceName, 'parameters.json');
```

Comment:
use `stateManager.getResourceParametersJson`

Reply:
I left it as is since this cod isn't changed, lines 14-61, the commit hooks keeps reformatting the chunk so I figured I will just leave it, instead of fighting it at every commit